spec/asset_path_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'file_path'
+
+RSpec.describe Metalware::FilePath do
+  describe 'asset files' do
+    let :file_path { Metalware::FilePath }
+
+    it 'defines an asset template' do
+      expect(file_path.asset_template('rack')).to eq('/var/lib/metalware/repo/assets/rack.yaml')
```

Comment:
These lines are quite long, they should stick to max 80 characters. Something like:
```
path = /v/l/m/...
expect(file_path.asset_template('rack').to eq(path)
```